Makefile.include

```diff
@@ -649,11 +649,13 @@ endef
 
 # Check if setsid command is available on the system for debug target
 # This is not the case on MacOSX, so it must be built on the fly
-ifneq (,$(filter debug, $(MAKECMDGOALS)))
-  ifneq (0,$(shell which setsid 2>&1 > /dev/null ; echo $$?))
-    SETSID = $(RIOTTOOLS)/setsid/setsid
-    $(call target-export-variables,debug,$(SETSID))
-    DEBUGDEPS += $(SETSID)
+ifeq ($(OS),Darwin)
+  ifneq (,$(filter debug, $(MAKECMDGOALS)))
```

Comment:
```suggestion
  ifneq (,$(filter debug%, $(MAKECMDGOALS)))
```

Actually just saw this

Reply:
The original did not have it either. Should I add it as a separate commit? If so, I need a reasoning ;-)

Reply:
Oh no never mind, `SETSID` is only used my `do_debug`m thanks for insisting.